BedBrigade/Client/wwwroot/index.html

```diff
@@ -14,6 +14,9 @@
     <link href="css/bedBrigade/navbar-top-fixed.css" rel="stylesheet" />
     <link rel="icon" type="image/png" href="favicon.png" />
     <link href="BedBrigade.Client.styles.css" rel="stylesheet" />
+    <link href="https://cdn.syncfusion.com/blazor/20.1.55/styles/bootstrap5.css" rel="stylesheet" />
```

Comment:
looks like we're adding bootstrap and bootstrap.js twice. If the syncfusion bootstrap is an extension which includes all the old styling / function then can we use them as drop in replacements? If so, we should remove the old css / js

Reply:
I've never looked to see if Syncfusion is the complete bootstrap or just overlays. I just don't know.